problem_builder/mentoring.py

```diff
@@ -355,12 +355,27 @@ def allowed_nested_blocks(self):
         except ImportError:
             pass
 
+        message_block_shims = [
+            NestedXBlockSpec(
+                MentoringMessageBlock,
+                category='pb-message',
+                boilerplate=message_type,
+                label=get_message_label(message_type),
```

Comment:
@antoviaque If two messages of the same type are added, there is an explicit error message on the parent (mentoring) block, although it's only visible after you update the page. It's not a big problem, so I think it would be fine to leave it for when Studio/xblock-utils support the use case.